app/views/publishers/vacancies/job_applications/reference_requests/show.html.slim

```diff
@@ -9,18 +9,28 @@
     h1.govuk-heading-l = @referee.job_application.name
 
     .govuk-body
-      - status = reference_request_status(@reference_request, @referee.job_reference)
+      - status = @reference_request.status
```

Comment:
As we are now ignoring 'marked as complete', I am fairly certain this has a high probability of introducing a regression, especially as the test for the deleted method has been deleted